content/news/2020-12-01-about-me.md

```diff
@@ -0,0 +1,74 @@
+title: All about my Outreachy internship with Mixxx DJ: Intro
+authors: Aanyu Deborah Oduman
+tags: website, Outreachy, internhip
```

Comment:
```suggestion
tags: personal, outreachy, internship
```

Let's use lowercase tags. `website` is a bit redundant because the blogpost doesn't actually has to do with the website.